### ISSUE TITLE
my manuskript is not launching i don't know why

### ISSUE THREAD

**Issue by Phoenix-BR** [olivierkes/manuskript#1207]
i tried to open in cmd to see the errors below is what said in the cmd

C:\Users\arthu\OneDrive\Área de Trabalho\manuskript>manuskript.exe
Logging to STDERR.
Logging to file: C:/Users/arthu/AppData/Local/manuskript/manuskript\logs\2023-08-25_18-52-50_manuskript#20704.log
Operating System: Windows-10-10.0.19045-SP0
Hardware: AMD64 / AMD64 Family 23 Model 1 Stepping 1, AuthenticAMD
Running in a frozen (packaged) state.
* sys.frozen = True
* sys._MEIPASS = C:\Users\arthu\OneDrive\Área de Trabalho\manuskript
* sys.executable = 'C:\\Users\\arthu\\OneDrive\\Área de Trabalho\\manuskript\\manuskript.exe'
* sys.argv = ['manuskript.exe']
* sys.path = ['C:\\Users\\arthu\\OneDrive\\Área de Trabalho\\manuskript\\base_library.zip',
 'C:\\Users\\arthu\\OneDrive\\Área de Trabalho\\manuskript\\..',
 'C:\\Users\\arthu\\OneDrive\\Área de Trabalho\\manuskript\\lib-dynload',
 'C:\\Users\\arthu\\OneDrive\\Área de Trabalho\\manuskript']
* sys.prefix = 'C:\\Users\\arthu\\OneDrive\\Área de Trabalho\\manuskript'
Manuskript 0.14.0 (Python 3.8.10)
* PyQt 5.15.6 (compiled against Qt 5.15.2)
  * Qt 5.15.2 (runtime)
* lxml.etree 4.9.0.0
  * libxml   2.9.12 (compiled: 2.9.12)
  * libxslt  1.1.34 (compiled: 1.1.34)
* pyEnchant 3.2.2 (libenchant: 2.2.7)
* pySpellChecker 0.6.3
* Symspellpy N/A
* Markdown 3.3.7
Web rendering engine: QTextEdit
Preferred translation: manuskript_pt_BR.qm (based on user setting)
Loaded translation: manuskript_pt_BR.qm
An unhandled exception has occurred!
Traceback (most recent call last):
  File "manuskript\ui\welcome.py", line 88, in showEvent
  File "manuskript\mainWindow.py", line 614, in loadProject
  File "manuskript\mainWindow.py", line 900, in loadDatas
  File "manuskript\loadSave.py", line 60, in loadProject
ValueError: invalid literal for int() with base 10: '\x00'
CRITICAL> An unhandled exception has occurred!
Traceback (most recent call last):
  File "manuskript\ui\welcome.py", line 88, in showEvent
  File "manuskript\mainWindow.py", line 614, in loadProject
  File "manuskript\mainWindow.py", line 900, in loadDatas
  File "manuskript\loadSave.py", line 60, in loadProject
ValueError: invalid literal for int() with base 10: '\x00'

**Comment by TheShadowOfHassen** [comment]
Did you set it to automatically load a project on start up?

**Comment by Phoenix-BR** [comment]
> Did you set it to automatically load a project on start up?

yep

**Comment by TheJackiMonster** [comment]
I think the project you are trying to load is broken. Was it a single-file project or stored as directory?

**Comment by Phoenix-BR** [comment]
i saved the project in my desktop like this and my manuscript there too
![Captura de Tela (319)](https://github.com/olivierkes/manuskript/assets/56056400/878655cb-1780-42b0-8c99-4efc3384ab96)

**Comment by TheJackiMonster** [comment]
Okay, so it's not a single file. That's good in case the project might be broken. The problem seems similar as in #1180

**Comment by Phoenix-BR** [comment]
Well, thank you for the help. I managed to get it working again. I just renamed the .msk and the folder as you said in the other issue. Then, I tried to import all the files from the old project by copying them and pasting them into the new folder I created for the project. All the files were there, with none corrupted at all. I guess the .msk file was corrupted. Thank you for helping me figure out what I needed to do :)
